### ISSUE TITLE
Error: cannot find -lgpt2

### ISSUE THREAD

**Issue by mkellerman** [go-skynet/LocalAI#52]
```
...
go mod edit -replace github.com/go-skynet/go-llama.cpp=/workspaces/LocalAI/go-llama
go mod edit -replace github.com/go-skynet/go-gpt4all-j.cpp=/workspaces/LocalAI/go-gpt4all-j
go mod edit -replace github.com/go-skynet/go-gpt2.cpp=/workspaces/LocalAI/go-gpt2
go run ./ api
# github.com/go-skynet/LocalAI
/usr/local/go/pkg/tool/linux_arm64/link: running g++ failed: exit status 1
/usr/bin/ld: cannot find -lgpt2
/usr/bin/ld: cannot find -lgpt2
/usr/bin/ld: cannot find -lgptj
collect2: error: ld returned 1 exit status

make: *** [Makefile:88: run] Error 1
```

Used #51 this makefile to generate my build on a macOS M1 arm machine.

**Comment by mkellerman** [comment]
i tried `make build` and `make buildgeneric`

**Comment by mkellerman** [comment]
fixed in #51
